downloads/swan-lake-release-notes/swan-lake-2201.5.0 /RELEASE_NOTE.md

```diff
@@ -93,6 +93,21 @@ If you have not installed Ballerina, download the [installers](/downloads/#swanl
 
 ### New features
 
+#### Support for regular expressions as a first-class element
+
+Ballerina now provides developers with a dedicated built-in type for regular expressions, enabling powerful pattern-matching and text processing operations in their code. The new type, named `RegExp`, is defined in the `lang.regexp` module and conforms to a subset of the ECMAScript specification for regular expressions.
+
+```ballerina
+import ballerina/lang.regexp;
+import ballerina/io;
+
+public function main() returns error? {
+    string:RegExp reg = re `[bB].tt[a-z]*`;
```

Comment:
Should we mention `string:RegExp` somewhere in the description to avoid confusion? Currently we've just said it's defined in `lang.regexp`.

Reply:
Updated, Please check

Reply:
What if we use `regexp:RegExp` as CEP in the example? Rephrasing seems more confusing to me. Actual details will be anyway available in the regexp doc refs

Reply:
My only concern with that is if it may give uses the idea that they have to use `regexp:RegExp`, but we prefer `string:RegExp`, right?

How about something like

> The new type, named `RegExp`, is defined in the new `lang.regexp` module. The `lang.string` module also contains an alias named `RegExp` for the same.

Reply:
Yes true. My only concern is that we don't actually need that information in the release note; isn't it? 
A clear example and a description would suffice. wdyt?

Reply:
Let's go ahead with

> Ballerina now provides a built-in type for regular expressions, enabling powerful pattern-matching and text processing operations in their code. The `RegExp` type is defined in the `lang.regexp` module, and can also be referenced using the type alias in the `lang.string` module. The `RegExp` type conforms to a subset of the ECMAScript specification for regular expressions.

Reply:
Updated